### PR TITLE
Added order Fulfillments details

### PR DESCRIPTION
**Pull request by RanAbram**
https://help.shopify.com/api/reference/order
https://help.shopify.com/api/reference/fulfillment

### CODE REVIEW

File: order.go [hunk 1]
```diff
@@ -114,19 +115,30 @@ type DiscountCode struct {
...
+	RequiresShipping           bool             `json:"requires_shipping"`
+	VariantInventoryManagement string           `json:"variant_inventory_management"`
+	Properties                 []NoteAttribute  `json:"properties"`
+	ProductExists              bool             `json:"product_exists"`
+	FulfillableQuantity        int              `json:"fulfillable_quantity"`
+	Grams                      int              `json:"grams"`
+	FulfillmentStatus          string           `json:"fulfillment_status"`
+	TaxLines                   []TaxLine        `json:"tax_lines"`
+	OriginLocation             Address          `json:"origin_location"`
+	DestinationLocation        Address          `json:"destination_location"`
```
Comment (dlebech) [inline order.go:141]: I can't find these two location fields in the docs. Where did you get them from?

File: order.go [hunk 2]
```diff
@@ -196,6 +208,28 @@ type Transaction struct {
 	PaymentDetails *PaymentDetails  `json:"payment_details"`
 }
 
+type Fulfillments struct {
```
Comment (dlebech) [inline order.go]: Could you change this to singular `Fulfillment`?

File: order.go [hunk 3]
```diff
@@ -196,6 +208,28 @@ type Transaction struct {
...
+	Service         string     `json:"service"`
+	UpdatedAt       *time.Time `json:"updated_at"`
+	TrackingCompany string     `json:"tracking_company"`
+	ShipmentStatus  string     `json:"shipment_status"`
+	TrackingNumber  string     `json:"tracking_number"`
+	TrackingNumbers []string   `json:"tracking_numbers"`
+	TrackingUrl     string     `json:"tracking_url"`
+	TrackingUrls    []string   `json:"tracking_urls"`
+	Receipt         Receipt    `json:"receipt"`
+	LineItems       []LineItem `json:"line_items"`
```
Comment (dlebech) [inline order.go:225]: Just a note: this field and a few others will probably be empty when viewing fulfillments via an `Order`. At least it seems that the the fulfillments on the order are very limited according to [the documentation](https://help.shopify.com/api/reference/order#fulfillments-property). This is just something to keep in mind for now.

**Review by dlebech — CHANGES_REQUESTED** [review]
Thanks for the PR @RanAbram :1st_place_medal: 

It's only the naming of the struct that needs changing, and then there's a small question as well.

Bonus points for adding a test to `order_test.go` for the fulfillments array though, but not essential here :-)

File: order.go (hunk 1 above)
Comment (RanAbram) [reply order.go:141]: That's the result i got when i tried to fetch my order:
screen shot 2018-01-06 at 20 29 53

File: order.go (hunk 3 above)
Comment (RanAbram) [reply order.go:225]: Cool, i'll keep it in mind when using those items. Thanks!

File: order.go (hunk 2 above)
Comment (RanAbram) [reply order.go]: @dlebech Done! 😃

**Comment by RanAbram** [comment]
@dlebech I fixed the comments you ask to. 💪

File: order.go (hunk 1 above)
Comment (dlebech) [reply order.go:141]: ok interesting. Thanks for confirming. It seems to be a common theme that the documentation is not complete regarding the fields it has.

**Review by dlebech — APPROVED** [review]
Thanks!

**Pull request merged**